src/generic/configure-modal/ConfigureModal.test.jsx

```diff
@@ -65,7 +65,7 @@ describe('<ConfigureModal /> for Section', () => {
     axiosMock = new MockAdapter(getAuthenticatedHttpClient());
   });
 
-  it('renders ConfigureModal component correctly', () => {
+  it.skip('renders ConfigureModal component correctly', () => {
```

Comment:
[question] Do we need to skip these tests?

Reply:
Not anymore, @ihor-romaniuk promptly found and fixed the problem in https://github.com/raccoongang/frontend-app-course-authoring/pull/152